include/picongpu/param/radiationObserver.param

```diff
@@ -124,5 +126,6 @@ namespace picongpu
 
     }
 
-  } // end namespace radiation_observer
-} // end namespace picongpu
+  } // namespace radiation_observer
```

Comment:
We don't indent `namespace`s, just the code in the deepest nested `namespace` with 4 spaces (see [here](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#6-namespaces)).

Reply:
I addressed the extra indentation